### DISCUSSION TITLE
Instagram Integration

### DISCUSSION THREAD

**Discussion by sebastianripari** [chatwoot/chatwoot#9764] [General]
Hello,

I am currently following the Instagram integration tutorial available [here](https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-channel-setup).

I am stuck at the step where I need to add **Instagram Basic Display**. Unfortunately, I don't see this option available in my setup. Could you please assist me in resolving this issue?

Thank you for your help!

Best regards

**Comment by laianeramos** [comment]
This option is only available on the app test, you have to creat an app test to select it.
